haiku-games/bebattle/bebattle-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="The BeBattle board game is a fun strategy game"
+DESCRIPTION="The BeBattle game is a fun board strategy game."
```

Comment:
BeBattle, a 1 or 2 players board strategy game in which you attempt to completly destroy your opponent's units.